TutorialMaker/Lib/TutorialGUI.py

```diff
@@ -274,14 +291,14 @@ def open_json_file_dialog(self):
             print("The user canceled the file dialog")
 
     def open_json_file(self, filepath):
-
+        
         directory_path = os.path.dirname(filepath)
         # Read the data from the file
-        with open(filepath, "r", encoding='utf-8') as file:
```

Comment:
There needs to be a consistent encoding used by all Tutorial Maker processes. Removing this opens the door to more problems related to Issue #22